src/test/spec/unified_runner/test_file.rs

```diff
@@ -102,6 +103,14 @@ impl RunOnRequirement {
                 return false;
             }
         }
+        if let Some(ref serverless) = self.serverless {
+            let is_serverless = SERVERLESS.as_ref().map_or(false, |s| s == "serverless");
```

Comment:
`Option<T>` only implements `PartialEq<Option<T>>`, so this little dance is needed to compare an `Option<String>` against an `Option<&str>` without a spurious temporary allocation.

Reply:
Personally I think it'd be more readable to do something along the lines of this, but I'm open to what others think.
```suggestion
            let is_serverless = if let Some(s) = SERVERLESS.as_ref() {
                s == "serverless"
            } else {
                false
            }
```

Reply:
Can we move this logic to the `lazy_static` definition? That way it can be used here and elsewhere simply as a boolean.

Reply:
Done.